### ISSUE TITLE
Support to get text without children

### ISSUE THREAD

**Issue by chengchengccc** [eeshi/node-scrapy#4]
I find a problem when I extract text from a node, text will includes text in all descendants.
however, I only need the text on this node, so I make a little enhancement which added a support to 
get text with out children of this node.

switch (item.get) {

```
  case 'text':

    get = function(node) { return item.prefix + trim.apply(node.text()) + item.suffix; }
    break;

  case 'textWithoutChildren':

    get = function(node) { return item.prefix + trim.apply(node.contents().filter(function() { return this.type === 'text'; }).text()) + item.suffix; }
    break;

  default:

    get = function(node) { return item.prefix + trim.apply(node.attr(item.get)) + item.suffix; }
    break;
}
```

**Comment by schoettler** [comment]
Hello gentlemen, i encountered a similar scenario:

Whenever node-scrapy finds, as shown in the example below, a `<p>` containing a `<b>` 
_for styling, or semantic purposes in case of `<strong>`_, 
scrapy will obtain the text once when scraping the `<p>` tag, and again when scraping the nested `<b>` tag.

Certain tags should be ignored,
 _or included in the JSON in case the developer wishes to keep the html semantics?_

![image](https://cloud.githubusercontent.com/assets/1758136/9321906/60a679a0-453a-11e5-94d4-ed282904cc2a.png)

**Comment by stefanmaric** [comment]
Available as the `$textNodes` getter since v0.4.0.

Given this document:

```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <title>Selector-less queries</title>
  </head>
  <body>
    <ul class="some-list">
      <li><span class="title">Name</span> James Heather</li>
      <li><span class="title">Sex</span> Male</li>
      <li><span class="title">Age</span> 46</li>
      <li>
        <span class="title">Resume</span> General Manager
        <p>2012, present as...</p>
      </li>
    </ul>
  </body>
</html>

```

And this model:

```json
{
  "items": [
    ".some-list > li",
    {
      "label": ".title",
      "value": "($textNodes | trim)"
    }
  ]
}
```

node-scrappy will return:

```json
{
  "items": [
    { "label": "Name", "value": "James Heather" },
    { "label": "Sex", "value": "Male" },
    { "label": "Age", "value": "46" },
    { "label": "Resume", "value": "General Manager" }
  ]
}
```
